app/presenters/check_answers/defendant_details_card.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module CheckAnswers
+  class DefendantDetailsCard < Base
+    attr_reader :defendant_details
+
+    KEY = 'defendant_summary'
+    GROUP = 'about_defendant'
+    def initialize(claim)
+      @defendant_details = claim.defendants
+      @group = GROUP
+      @section = KEY
+    end
+
+    def route_path
+      KEY
+    end
+
+    def rows
+      all_rows = main_defendant_rows
+      additional_defendants.each_with_index do |defendant, index|
+        all_rows.concat additional_defendant_rows(defendant, index + 1)
+      end
+      all_rows
+    end
+
+    private
+
+    def main_defendant
+      defendant_details.find { |defendant| defendant[:main] == true }
+    end
+
+    def additional_defendants
+      defendant_details.select { |defendant| defendant[:main] == false }
+    end
+
+    def main_defendant_rows
+      [
+        {
+          key: { text: translate_table_key(KEY, 'main_defendant_full_name'),
+classes: 'govuk-summary-list__value-width-50' },
+          value: { text: main_defendant[:full_name] }
+        },
+        {
+          key: { text: translate_table_key(KEY, 'main_defendant_maat'), classes: 'govuk-summary-list__value-width-50' },
```

Comment:
Where in the form we are adding Maat ID for defendants? not in the defendant add page for sure :/

Reply:
![Screenshot 2023-07-19 at 09 46 53](https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/84575729/4af3c7ce-9a16-4438-b529-a75f85c78b0f)

MAAT ID can be found here

Reply:
<img width="900" alt="image" src="https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/26544538/26d229c0-0baf-471b-affe-3422f1d8c7ec">

Reply:
Why its not showing here ?? :/
![Screenshot 2023-07-19 at 09 53 22](https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/90189839/5f60c875-574c-400c-bb15-ce1bc95482a0)

Reply:
[Add defendant](https://crm7dev.apps.live.cloud-platform.service.justice.gov.uk/applications/bd7845eb-44dd-4ef2-b5bf-bc207705e741/steps/defendant_details/00000000-0000-0000-0000-000000000000)

Reply:
I think that's a bug
<img width="794" alt="image" src="https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/26544538/4838d8b7-409f-47a3-943a-67e374bcb4d4">

Reply:
Got it. It seems like MAAT ID is not required if claim type in ' Breach of Injunction'. In that case, in check your answer page , Do we need a check whether to show MAAT ID or not?


    validates :maat, presence: true, if: :maat_required?
    def maat_required?
      application.claim_type != ClaimType::BREACH_OF_INJUNCTION.to_s
    end

Reply:
if this is optional input... should it also be optional display?

Reply:
identified this is due to the type of case and is not a bug

Reply:
This is not optional, if Breach of injunction we currently not rendering the MAAT ID at all, hence no option to enter.

Reply:
will rectify in next pr